util/diff/diff.go

```diff
@@ -0,0 +1,18 @@
+package diff
+
+import (
+	"encoding/json"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	log "github.com/sirupsen/logrus"
+)
+
+func LogChanges(old, new interface{}) {
+	if !log.IsLevelEnabled(log.DebugLevel) {
+		return
+	}
+	a, _ := json.Marshal(old)
+	b, _ := json.Marshal(new)
+	patch, _ := jsonpatch.CreateMergePatch(a, b)
+	log.Debug(string(patch))
```

Comment:
If we're going to keep debug statements, can we place add an identifier string to them? E.g.
```suggestion
	log.Debugf("Log changes patch: %s", string(patch))
```
This way if it shows up in the logs its easy to find

Reply:
Also, very minor, but I would consider having this function to return the string and for the caller of the function to do the logging.